<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerSettingWidget</class>
 <widget class="QDialog" name="TriggerSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据采集-设置采集模式</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/gui/icons.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/gui/icons.qrc">
       <normaloff>:/image/cross.png</normaloff>:/image/cross.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/gui/icons.qrc">
       <normaloff>:/image/tick.png</normaloff>:/image/tick.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/gui/icons.qrc">
       <normaloff>:/image/reset.png</normaloff>:/image/reset.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>曝光时间：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>触发模式：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>手动进出洞控制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自动进出洞控制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自由触发模式</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>脉冲间隔：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>0.5m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.75m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.25m</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>相机采集模式：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>连续采集</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>硬触发</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>软件同步触发</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/gui/icons.qrc"/>
 </resources>
 <connections/>
</ui>
